@middy/db-manager/index "has no call signatures"

What are the steps to reproduce this issue?
-------------------------------------------
1. Import `db-manager` using typescript
```import dbManager from '@middy/db-manager';```
2. Try to call the db manager function
```
exports.handler = middy(handler)
.use(httpErrorHandler())
.use(jsonBodyParser())
.use(validator({ inputSchema: inputSchema }))
.use(
    dbManager({ // this is where it fails
        config: dbConfig,
    })
);
```

What happens?
-------------
Receive error saying db-manager/index has no call signatures.

What were you expecting to happen?
----------------------------------
I was expecting the db-manager function to operate like other pieces of this middleware. jsonBodyParser is working just fine, for example.

Any logs, error output, etc?
----------------------------
Test suite failed to run
```
    src/myFile/index.ts:16:9 - error TS2349: This expression is not callable.
      Type 'typeof import("/path/to/my/project/node_modules/@middy/db-manager/index")' has no call signatures.

    16         dbManager({
               ~~~~~~~~~
 FAIL  src/myFile/myTest.test.ts
  ● Test suite failed to run
```


What versions of software are you using?
----------------------------------------
**Typescript Version:** 3.2.4

**Middy Version:** 1.0.0

**Serverless:** 1.70.1

--- a/packages/db-manager/index.d.ts
+++ b/packages/db-manager/index.d.ts
@@ -1,3 +1,4 @@
+import middy from '@middy/core';
 import Knex from 'knex';
 
 export interface DbManagerOptions {
@@ -9,3 +10,7 @@ export interface DbManagerOptions {
   secretsParam?: string,
   secretsPath?: string
 }
+
+declare const dbManager: middy.Middleware<DbManagerOptions, any, any>
+
+export default dbManager

